test/NuGet.Core.Tests/NuGet.Build.Tasks.Pack.Test/PackTaskLogicTests.cs

```diff
@@ -13,12 +13,35 @@
 
 namespace NuGet.Build.Tasks.Pack.Test
 {
+    /*
+       This test uses the ...\NuGet.Build.Tasks.Pack.Test\compiler\resources\project.assets.json assets file.
```

Comment:
This PR deletes this file. Is that intentional?

Reply:
Its technically renamed but it was edited so much that git sees it as a delete and add.